tgui/packages/tgui/interfaces/OperatingComputer.js

```diff
@@ -0,0 +1,534 @@
+
+import { useBackend, useLocalState } from '../backend';
+import { Box, ColorBox, Chart, Section, Stack, Tabs, Table } from '../components';
+import { Window } from '../layouts';
+import { HealthStat } from './common/HealthStat';
+import { COLORS } from '../constants';
+import { ReagentGraph } from './common/ReagentInfo';
+import { getStatsMax, processStatsData } from './EngineStats';
+import { capitalize, spaceUnderscores } from './common/stringUtils';
+
+export const OperatingComputer = (props, context) => {
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 1);
+
+  return (
+    <Window title="Operating Computer" width="560" height="760">
+      <Window.Content scrollable>
+        <Tabs>
+          <Tabs.Tab
+            selected={tabIndex === 1}
+            onClick={() => setTabIndex(1)}>
+            Patient Health
+          </Tabs.Tab>
+          {/* <Tabs.Tab
+            selected={tabIndex === 2}
+            onClick={() => setTabIndex(2)}>
+            Emergency Medical Helper
+          </Tabs.Tab> */}
+        </Tabs>
+        <ComputerTabs tabIndex={tabIndex} />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const ComputerTabs = (props) => {
+  const { tabIndex } = props;
+  if (tabIndex === 1) { return (<PatientTab />); }
+};
+
+// mob.stat & crit parsing
+const PatientSummary = (props) => {
+  const { occupied, patient_status, health } = props;
```

Comment:
have them inherit a base for occupied status, they pretty much all want to use it?